### DISCUSSION TITLE
Have question about .pkg signing

### DISCUSSION THREAD

**Discussion by KevinLiAWS** [aws/aws-cli#7362] [Q&A] [answered]
I contacted with AWS Signer team, and they said AWS-CLI is onboarding Windows with them, but they don't support .pkg signing for now. Would like to ask currently how AWS-CLI .pkg get signed? Thank you.

**Comment by RyanFitzSimmonsAK** [comment, marked as answer]
This question was addressed internally, please reach out to the team directly for more specifics going forward.
